### ISSUE TITLE
Add shell completion for UV in feature

### ISSUE THREAD

**Issue by lucasfijen** [va-h/devcontainers-features#4]
Hey! Thank you for your nice feature for UV in devcontainers!

I was wondering if it would be possible to add the shell completion scripts to your feature, so as per [documentation](https://docs.astral.sh/uv/getting-started/installation/#shell-autocompletion).

That would include running:
```bash
# Determine your shell (e.g., with `echo $SHELL`), then run one of:
echo 'eval "$(uv generate-shell-completion bash)"' >> ~/.bashrc
echo 'eval "$(uv generate-shell-completion zsh)"' >> ~/.zshrc
echo 'uv generate-shell-completion fish | source' >> ~/.config/fish/config.fish
echo 'eval (uv generate-shell-completion elvish | slurp)' >> ~/.elvish/rc.elv

# Determine your shell (e.g., with `echo $SHELL`), then run one of:
echo 'eval "$(uvx --generate-shell-completion bash)"' >> ~/.bashrc
echo 'eval "$(uvx --generate-shell-completion zsh)"' >> ~/.zshrc
echo 'uvx --generate-shell-completion fish | source' >> ~/.config/fish/config.fish
echo 'eval (uvx --generate-shell-completion elvish | slurp)' >> ~/.elvish/rc.elv


```

**Comment by va-h** [comment]
Hi Lucas,

Thank you for the request, It took a bit longer since I wanted to remove the devcontainer-contrib dependency first, but I implemented it today in 749335c, with the variable "shellautocompletion" set to true Autocompletion should work. For the moment I will default to disabled until I have tested it a week or two myself during day to day.

Best regards
Valentin

**Comment by lucasfijen** [comment]
Thank you very much! At first sight it seems to work great! I'll let you know if i run into any issues, but so far it seems to work great!
